_includes/tutorials/fk-joins/kstreams/code/tutorial-steps/dev/console-producer-albums.sh

```diff
@@ -0,0 +1,5 @@
+docker exec -i schema-registry /usr/bin/kafka-avro-console-producer --topic albums --broker-list broker:9092\
+  --property "parse.key=true"\
```

Comment:
as of the 5.4 release, the `kafka-avro-console-producer` accepts a `key-schema` parameter.  I've validated this works locally.